share/completions/nvram.fish

```diff
@@ -0,0 +1,12 @@
+# completion for nvram (macOS)
+
+function __fish_nvram_variables
+    command nvram -p
```

Comment:
OK, it seems a little bit unnecessary to me still, but I've removed the change request.